articles/flows/guides/auth-code-pkce/includes/authorize-user-call-api.md

```diff
@@ -26,7 +26,7 @@ https://${account.namespace}/authorize?
 
 #### Parameters
 
-Note that for authorizing a user when calling an API, you:
+Note that for authorizing a user when calling a custom API, you:
```

Comment:
The docs use the word "custom" to refer to an API that the user is building, as opposed to the Management API or similar. I just updated the wording in the auth-code and auth-code-pkce flows to match what we're doing in the device code flow since it seemed a similar objection could be had there as well.